File: Q19/Q19.py
def find_longest_word(sentence):
    words = sentence.split()
    longest_word = ''
    for word in words:
        if len(word) > len(longest_word):
            longest_word = word
    return longest_word

sentence = "Find the longest word in a string"
print(find_longest_word(sentence))  

File: Q19/Q19.ts
function findLongestWord(str: string): string {
  let words = str.split(" ");
  let longestWord = "";
  for (let word of words) {
    if (word.length > longestWord.length) {
      longestWord = word;
    }
  }
  return longestWord;
}

const sentence = "Find the longest word in a string";
console.log(findLongestWord(sentence));
